.vscode/launch.json

```diff
@@ -0,0 +1,27 @@
+{
+  // Use IntelliSense to find out which attributes exist for node debugging
+  // Use hover for the description of the existing attributes
+  // For further information visit https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Launch Program",
+      "type": "node2",
+      "request": "launch",
+      "program": "${workspaceRoot}/server/dist/server.js",
+      "cwd": "${workspaceRoot}",
+      "sourceMaps": true,
+      "outFiles": [
+        "${workspaceRoot}/server/dist"
+      ]
+    },
+    {
+      "name": "Attach to Process",
+      "type": "node2",
+      "request": "attach",
+      "port": 5858,
+      "outFiles": [],
+      "sourceMaps": true
+    }
+  ]
+}
```

Comment:
I use npm install --save so npm adds the deps. I used mostly because it allows me to use double quotes in Strings without escaping. Maybe we need some Style guide anderen linting tool

I try to consider the other things in the future